documentation/modules/ROOT/examples/either-declaration.ttl

```diff
@@ -13,22 +13,26 @@
 @prefix unit: <urn:samm:org.eclipse.esmf.samm:unit:2.0.0#> .
 @prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
 
+:AspectWithEither a samm:Aspect ;
+  samm:properties ( :speedProperty ) ;
+  samm:operations ( ) .
+
 # tag::either-declaration[]
-:Result a samm-c:Either ;
-   samm-c:left :ErrorCode ;
-   samm-c:right :SuccessCode .
+:speedProperty a samm:Property ;
+  samm:characteristic :speedValue .
+
+:speedValue a samm-c:Either ;
```

Comment:
To conform to naming rules, the Characteristic name should be upper case, so I suggest we keep the name "Result", since it's also still/already referred to by this in payloads.adoc:

```suggestion
  samm:characteristic :Result .

:Result a samm-c:Either ;
```